packages/teleport/src/Sessions/SessionList/SessionList.tsx

```diff
@@ -98,3 +111,21 @@ type Props = {
   sessions: Session[];
   pageSize?: number;
 };
+
+function participantMatcher(
+  targetValue: any,
+  searchValue: string,
+  propName: keyof Session & string
+) {
+  if (propName === 'parties') {
+    return targetValue.some((participant: Participant) => {
+      if (participant.remoteAddr.toLocaleUpperCase().includes(searchValue)) {
```

Comment:
on second thought, idk if we should be relying on the root searchValue transofrmation b/c someone can change it and then everything will break right? Lets also transform the search value in our matchers, and then in the root match only transform when about to match?